corehq/form_processor/utils.py

```diff
@@ -33,3 +35,12 @@ def _inner(*args, **kwargs):
         return _wrap(obj)
 
     return _inner
+
+
+def should_use_sql_backend(domain):
+    toggle_to_check = USE_SQL_BACKEND
```

Comment:
this is just the toggle so that we can turn the postgres db on at a per domain basis. also this at the moment doesn't need to be a variable, but once it becomes a random toggle we'll need to make sure the randomness is 0 in the tests